setup.py

```diff
@@ -32,7 +32,7 @@ def read_file(filename, encoding='utf8'):
 
 install_requires = [
     'acme=={0}'.format(version),
-    'ConfigArgParse',
+    'ConfigArgParse>=0.10.0',  # python2.6 support, upstream #17
```

Comment:
Version pin should be moved to else branch of sys.version_info < (2, 7) condition checked below.